app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -216,6 +217,8 @@ class SupplementalApplicationPage extends React.Component {
       rentalAssistance,
       this.state.persistedApplication.id
     )
+    // show price in right format
+    rentalAssistance.assistance_amount = formUtils.formatPrice(rentalAssistance.assistance_amount)
```

Comment:
@software-project I wonder if we could get this line covered with a snapshot test that verifies that currencies are formatted with $, etc.

Reply:
It's already covered in rental-assistance e2e test. I modified it just now to also test for format with comma.